packages/xstate-store/src/types.ts

```diff
@@ -382,14 +384,18 @@ export interface AtomOptions<T> {
   compare?: (prev: T, next: T) => boolean;
 }
 
+export interface AsyncAtomOptions<T> extends AtomOptions<T> {
+  signal?: AbortSignal;
```

Comment:
This was only added to the type here - but it's not handled by the implementation and there are no tests for this